common/src/test/groovy/com/mx/path/core/common/reflection/AnnotationsTest.groovy

```diff
@@ -22,21 +22,29 @@ class AnnotationsTest extends Specification {
     def fields = Annotations.fieldsWithAnnotation(Nullable.class, WithAnnotations.class)
 
     then:
-    fields.size() == 1
-    fields[0].getField().getName() == "aField"
+    fields.size() == 3
+    fields[0].field.name == "aField"
+    fields[1].field.name == "baseField"
+    fields[2].field.name == "baseBaseField"
   }
 
   def "fieldsWithAnnotations"() {
     when:
-    def fields = Annotations.fieldsWithAnnotations(WithAnnotations.class)
+    def fields = Annotations.fieldsAndAnnotations(WithAnnotations.class)
```

Comment:
Won't we still have a `fieldsWithAnnotation`. Does that need to keep it's tests?

Reply:
fieldsWithAnnotations just calls fieldsAndAnnotations(class). It's only around for backward compatibility.